static/cdn/maintenance.html

```diff
@@ -1,208 +1,329 @@
+				height: 66px; /*height of image*/
+				width: 63px;  /*width of image*/
+				position: absolute;
+				background-size: 24px 24px;
+				background-repeat: no-repeat;
+			}
+			h2.comment:before {
+				margin-top: 0;
+				margin-left: -2.375rem;
+				content:'';
+				background: url('http://scratch-maintenance.s3.amazonaws.com/comment.svg');
+				height: 66px; /*height of image*/
+				width: 63px;  /*width of image*/
+				position: absolute;
+				background-size: 25px 24px;
+				background-repeat: no-repeat;
+			}
+			p {
+				font-size: 1rem;
+				line-height: 1.5rem;
+				margin: 1rem 1rem 1.5rem 1rem;
+			}
+			.narrow {
+				padding: 0 10%;
+			}
+			button {
+				cursor: pointer;
+				border: 0;
+				background-color: white;
+				box-shadow: none;
+				border-radius: .25rem;
+				padding: 1rem 1.5rem;
+			}
+			button span {
+				font-size: 1rem;
+				color: #4D97FF;
+				font-weight: 600;
+			}
+
+			.hidden {
+				display: none;
+			}
+			.showing {
+				display: inline-block;
+			}
+			.grid-wrapper {
+				min-height: 100%;
+				/* display: flex; */
+				margin-bottom: 2rem;
+			}
+			.grid-content {
+				/* display: inline-block; */
+			}
+			.grid-content .grid-box {
+				width: 16rem;
+				display: inline-block;
+				background-color: white;
+				font-size: 1rem;
+				color: #4D97FF;
+				font-weight: 600;
+				border-radius: .5rem;
+				padding: 1rem 1rem;
+				margin: 0.5rem;
+				text-align: left;
+				vertical-align: top;
+			}
+			.os-name-row {
+				margin-bottom: 0.25rem;
+				display: flex;
+				align-items: center;
+			}
+			.os-name-img {
+			}
+			.os-name-text {
+				color: #575E75;
+				line-height: 1.25rem;
+				margin-left: 0.5rem;
+			}
+			.os-version {
+				color: #aaa;
+				font-size: 0.75rem;
+				font-weight: 400;
+			}
+			.download-link-row {
+				margin-top: 0.75rem;
+				display: flex;
+				align-items: center;
+			}
+			.store-img {
+				flex: 1;
+				height: 2.625rem;
+			}
+			.store-img img {
+				height: 2.5rem;
+			}
+			.vertical-separator-left {
+				border-right: 1px solid #d9d9d9;
+				height: 2rem;
+				flex: 1;
+			}
+			.vertical-separator-right {
+				flex: 1;
+			}
+			.download-link {
+				flex: 1;
+				padding: .25rem 0.125rem .25rem 0;
+			}
+			.download-link a {
+				color: #4D97FF;
+				text-decoration: none;
+			}
+		</style>
+		<script language="JavaScript">
+      var toggle = function (elem) {
+      	// If the element is visible, hide it
+      	if (window.getComputedStyle(elem).display === 'block') {
+        	elem.style.display = 'none';
+      		return;
+	      }
+      	// Otherwise, show it
+      	elem.style.display = 'inline-block';
+      };
+			// Handler when the DOM is fully loaded
+			var onLoaded = function() {
+				// Listen for click events
+				var actionShowElements = document.getElementsByClassName('toggleShowing');
+				if (actionShowElements && actionShowElements.length) {
+					for (var i = 0; i < actionShowElements.length; i++) {
```

Comment:
Very possible I'm misunderstanding this, but it looks like ".toggleShowing", ".showing", and ".hiding" are on, at max, one element each. If so, I'm wondering if we could do this toggling without the for loops.

Reply:
Totally. I wanted to take the opportunity to learn more about how this sort of basic DOM manipulation works with Javascript. Kinda overkill!

Reply:
Gotcha, makes sense!